App.js

```diff
@@ -0,0 +1,118 @@
+import React, {useEffect, useState} from 'react';
+import { StyleSheet, Text, View, Platform } from 'react-native';
+import Geolocation from 'react-native-geolocation-service';
```

Comment:
This was flagged by DeepScan since it's not being used. Is this needed by the other imports?